srcpkgs/naken_asm/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'naken_asm'
+pkgname=naken_asm
+version=20190317
+revision=1
+wrksrc=${pkgname}-2019-03-17
+build_style=gnu-configure
```

Comment:
```suggestion
build_style=configure
```
this is a custom configure script, not a gnu one. You also need to add a few args to `configure_args=` to fix installation of the files and cross cimpiling.

Reply:
It is recommended that you actually test your packages locally. A empty package is never a good sign...

Reply:
Currently adjusted the template as you said but the package gets created only on i686: on the other architectures it stops at the building phase.

Reply:
> Currently adjusted the template as you said but the package gets created only on i686: on the other architectures it stops at the building phase.

Actually the i686 build only finished because of a bug in xbps-src that it doesn't detect that the package is empty.

Reply:
Oh thanks for letting me know that! I will try to resolve the problem.